dashboard/cron.py

```diff
@@ -128,25 +128,29 @@ def update_user(self):
         # loop through multiple course ids
         for data_warehouse_course_id in Course.objects.get_supported_courses():
 
-            # select all student registered for the course
-            user_sql = f"""select u.name AS name,
-                        p.sis_user_id AS sis_id,
-                        p.unique_name AS sis_name,
-                        u.global_canvas_id AS id,
-                        c.current_score AS current_grade,
-                        c.final_score AS final_grade,
-                        '{data_warehouse_course_id}' as course_id
-                        from user_dim u,
-                        pseudonym_dim p,
-                        course_score_fact c,
-                        (select e.user_id as user_id, e.id as enrollment_id from enrollment_dim e
-                        where e.course_id = '{data_warehouse_course_id}'
-                        and e.type='StudentEnrollment'
-                        and e.workflow_state='active' ) as e
-                        where p.user_id=u.id
-                        and u.id = e.user_id
-                        and c.enrollment_id =  e.enrollment_id
-                        and p.sis_user_id is not null
+            # select all student, tas, and instructors registered for the course
+            user_sql = f"""
+                        with enrollment as (
```

Comment:
There was [commit](https://github.com/tl-its-umich-edu/my-learning-analytics/commit/8405cd2beedc70e168484683909c9846996b0f99) that made, I don't see that this PR is picking up that changes. The commit was based of this [issue](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/407),those changes need to be included as well